### ISSUE TITLE
Inline literals should use {{ }} instead of `` ``

### ISSUE THREAD

**Issue by mchataigner** [tonybaloney/sphinxcontrib-confluencebuilder#56]
Hello,
Currently if I have a literal in my markup using \`\`my_literal\`\` the result in confluence is \`\`my_literal\`\`.
But confluence has support for literals with {{ }}, so instead, it should generate {{my_literal}}.

Thanks.

**Comment by tonybaloney** [comment]
Yes, confirmed https://confluence.atlassian.com/doc/confluence-wiki-markup-251003035.html

**Comment by jdknight** [comment]
> if I have a literal in my markup using \`\`my_literal\`\` the result in confluence is \`\`my_literal\`\`.

The current release of this extension (v0.6) does not properly render inline literals; however, the current master does resolve this situation. See also:

- [inline literal v0.6](https://jdknight.atlassian.net/wiki/spaces/confluencebuilder/pages/322773/v0.6-inline+markup) (broken)
- [inline literal develop](https://jdknight.atlassian.net/wiki/spaces/confluencebuilder/pages/45744787/devel-inline+markup) (working)

This issue should be resolved in #29 which should be done really really soon (ideally before or during the first week of December '17).

I'll close this off when #29 is assessed, unless there is another issue related to rendering inline literals you want to bring up?

**Comment by jdknight** [comment]
@mchataigner, v0.7 has been released -- want to try with the newer version to see if you get expected results?

**Comment by mchataigner** [comment]
For some reason, it does not work correctly if it's inside a list, i.e.:
* toto ``literal`` tata
will render like that:
- toto
  {{
      literal
  }}
  tata

**Comment by jdknight** [comment]
@mchataigner, any chance you can make a snippet of the raw file which causes this issue?

----

I just tried with v0.7 with the following appended to a file:

```
- This is a bullet list with *emphasized text*, **strong text**,
  `interpreted text` and ``inline literals``.

* toto ``literal`` tata
```
Which yielded "expected" formatting:

![image](https://user-images.githubusercontent.com/1834509/33590742-4e841f6a-d94f-11e7-906a-0e61aa46b7d1.png)

**Comment by jdknight** [comment]
@mchataigner, I've been able to find a scenario where your unexpected results appear.

If we add the inlined literals inside a block quote, the undesired render effect appears. For example:

```

- This is a bullet list with *emphasized text*, **strong text**,
  `interpreted text` and ``inline literals``.

* toto ``literal`` tata

space

    additional

    - This is a bullet list with *emphasized text*, **strong text**,
      `interpreted text` and ``inline literals``.

    * toto ``literal`` tata

end
```

Renders the following:

![image](https://user-images.githubusercontent.com/1834509/34510499-564b1ce6-f022-11e7-9884-b0ae6fd2c7c4.png)

This issue is actually addressed by a proposed fix (see PR #72; commit af5810076a78d1a51e7ed2c222a057b6e0c03b59).

----

Since this initial issue is focused on the inline literal markup which has been addressed in v0.7 release, I'm going to mark this issue as closed. I would expect that your additional formatting issues you've observed will be addressed in the mentioned pull request and will be available soon on `master`, followed by a future v0.9 release. Thanks for reporting the issues you've observed.
